packages/front/src/services/api.ts

```diff
@@ -1,3 +1,4 @@
+import { useEnv } from "@/hooks/useEnv";
 import axios from "axios";
 
 export const api = axios.create({
```

Comment:
This const api is not used anywhere in the app. Looks like it referred to the previous version of the app, which had a digital ocean relayer